src/Config/CS_SDK.props

```diff
@@ -22,6 +22,7 @@
     <NoWarn>MSB3539</NoWarn><!--The value of the property "BaseIntermediateOutputPath" was modified after it was used by MSBuild-->
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <DebugSymbols>true</DebugSymbols>
+    <SelfContained>false</SelfContained>
```

Comment:
`SelfContained` changes default values based on presence of other flags.
Explicitly setting to false guarantees the value.

Reply:
I was going by this conversation
https://github.com/dotnet/sdk/issues/2505#issuecomment-417404229